javascript/javascript3/week3/circleCreationArt.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+let canvas = document.getElementById("canvas");
+let ctx = canvas.getContext("2d");
+
+//making canvas size same as window
+let window_width = window.innerWidth;
+let window_height = window.innerHeight;
+
+canvas.width = window_width;
+canvas.height = window_height;
+let width = canvas.width;
+let height = canvas.height;
+setInterval(() => {
+  function randomCircle() {
+    let randomx = Math.random() * width;
+    let randomy = Math.random() * height;
+    let randomr = Math.random() * 100;
+    let r = Math.random() * 255;
```

Comment:
same code repeated 3 times. You could extract it into a function `getRandomColor()`